client/src/index.html

```diff
@@ -114,6 +114,10 @@
 <!--locale-->
 <script src="vendor/angular-dynamic-locale/dist/tmhDynamicLocale.js"></script>
 
+<!-- ng-file-upload -->
+<script src="vendor/ng-file-upload-shim/ng-file-upload-shim.min.js"></script> <!-- for no html5 browsers support -->
```

Comment:
I'm pretty sure we don't need this file -- although that decision can be made when we discuss installation.  At the moment, we only "officially" support the latest chrome, which supports HTML5.